t/unit/backends/test_database.py

```diff
@@ -550,6 +550,37 @@ def test_backend__task_result_meta_injection(self):
         tr = TaskResult.objects.get(task_id=tid2)
         assert json.loads(tr.meta) == {'key': 'value', 'children': []}
 
+    @pytest.mark.skip("Need to find how to inject stamped headers")
```

Comment:
mmmm.... not sure if we should merge this with this test only.... should we wait bit more?

Reply:
Yeah I agree this needs testing - I just couldn't work out how to actually go about setting the stamp on the request

Reply:
were you able to figure that out?